### ISSUE TITLE
unable to get past login screen, pop up says "you are logged in at <site name/> using your credentials from Site Admin"

### ISSUE THREAD

**Issue by gmsharpe** [instructure/canvas-lms#1587]
### Summary:
After replicating the 'production setup' in a Dockerfile, I'm able to successfully build and run a production server.  However, when I go to login, I can't get past the login screen.  I get an error like the following:  "you are logged in at <site name/> using your credentials from Site Admin."  This is just a pop up.  Looking at the network trace, it appears I've been re-directed to the home page.

I realize this is somewhat vague, but I'm hoping someone recognizes this behavior, and can point me in the right direction.

**Comment by simonista** [comment]
Hey Gary, so just to be clear, you:
1. go to <canvas-url>
2. get redirected to <canvas-url>/login/canvas
3. type your username/password, hit the button
4. end up back on <canvas-url>/login/canvas but with a little in-page notification that says ""you are logged in at using your credentials from Site Admin."

Did I get that right? And presumably if you then try to go directly to another page like just <canvas-url>, you get redirected back to the login page again?

**Comment by gmsharpe** [comment]
Yes, that summed it up, and I did get redirected back to the login page when trying to navigate anywhere else.  I managed to solve the issue yesterday afternoon.  The redis.yml file I was working with did not have an entry for a 'production' environment.  After adding one, I've been able to navigate beyond the login page.

**Comment by ccutrer** [comment]
Please use [the mailing list](https://groups.google.com/g/canvas-lms-users) for installation and operational issues.
